webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -537,7 +537,7 @@ export default {
         // Disable suggester on smart-phone landscape
         extraPlugins = 'simpleLink,selectImage';
       }
-      CKEDITOR.addCss('.cke_editable { font-size: 14pt; font-family: Helvetica, regular, sans-serif; }');
+      CKEDITOR.addCss('.cke_editable { font-size: 14pt; font-family: Helvetica, regular, sans-serif; } .video { width: 900px; height: 506px; margin: 0 auto; }');
```

Comment:
should this be in less file ?

Reply:
> should this be in less file ?
I want to change the height and width of the video that is displayed in ckeditor, that's why I did not add it in the less file I add them directly in ckeditor .

Reply:
and why it couldnt be in a less file ?

Reply:
Ok, this is a specific css for news